src/wrapper/qr_mumps_api.jl

```diff
@@ -347,6 +347,45 @@ for (fname, lname, elty, subty) in (("sqrm_spbackslash_c", libsqrm, Float32   ,
     end
 end
 
+for (fname, lname, elty, subty) in (("sqrm_spfct_backslash_c", libsqrm, Float32   , Float32),
+                                    ("dqrm_spfct_backslash_c", libdqrm, Float64   , Float64),
+                                    ("cqrm_spfct_backslash_c", libcqrm, ComplexF32, Float32),
+                                    ("zqrm_spfct_backslash_c", libzqrm, ComplexF64, Float64))
+    @eval begin
+        function qrm_spbackslash!(spfct :: qrm_spfct{$elty}, b :: Vector{$elty}, x :: Vector{$elty})
```

Comment:
`qrm_spfct_spbackslash!` ?

Reply:
@amontoison you make me realize that all of these qrm functions modify the b right-hand side. should we remove all the methods without ! or for all of them, make a temporary copy of b?

Reply:
For all the methods without ! we should make a temporary copy of b.
For all the methods with ! we just need to precise in the documentation that b is also updated.